pkg/cleanup/bridge_windows.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2021 k0s authors
```

Comment:
Bummer! I bet this is excluded because ob build tags. Maybe we should run the golangci-lint also for Windows?

Reply:
Makes sense, since you have a windows machine, would you mind running go lint in your host just to check it?

Reply:
OK I got different results by runnign GOOS=windows make lint.
Turns out there are a couple more issues with lint and windows. I'll fix this issue in a different PR and make sure that we run the lint for windows, linux and maybe darwin